### PR TITLE
Adding Food EndPoint

### PR DESCRIPTION
**Pull request by YahyaAiman**

### CODE REVIEW

File: src/services/food/handlers.ts [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+    return res.status(200).json({
+      message: responseMessages.success.delete,
+    });
+  } catch (error) {
+    next(error);
+  }
+};
+
+export const foodHandler = {
```
Comment (day280130) [inline src/services/food/handlers.ts]: should be `foodHandlers` instead of `foodHandler`

File: src/services/food/handlers.ts [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        message: responseMessages.error.reqParams,
+      });
+    }
+    const inputBody = foodSchemas.postBody.safeParse(req.body);
+    if (!inputBody.success) {
+      return res.status(400).json({
+        message: responseMessages.error.reqBody,
+      });
+    }
+    const updateResult = await prisma.food.update({
```
Comment (day280130) [inline src/services/food/handlers.ts]: please handle record not found error on the catch block below

File: src/services/food/handlers.ts [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+const deleteFood: ReqHandler = async (req, res, next) => {
+  try {
+    const paramId = foodSchemas.params.safeParse(req.params);
+    if (!paramId.success) {
+      return res.status(400).json({
+        message: responseMessages.error.reqParams,
+      });
+    }
+
+    const deleteResult = await prisma.food.delete({
```
Comment (day280130) [inline src/services/food/handlers.ts]: please handle record not found error on the catch block below

**Review by day280130 — CHANGES_REQUESTED** [review]
please keep consistency on variables name and handle errors properly

File: src/services/food/handlers.ts (hunk 1 above)
Comment (YahyaAiman) [reply src/services/food/handlers.ts]: OK, I'll change it

File: src/services/food/handlers.ts (hunk 2 above)
Comment (YahyaAiman) [reply src/services/food/handlers.ts]: ah that's right, it doesn't catch the error from prisma, i'll catch it on catch block

File: src/services/food/handlers.ts (hunk 3 above)
Comment (YahyaAiman) [reply src/services/food/handlers.ts]: Ahh, this is still the same as in the update handler

**Review by day280130 — APPROVED** [review]
good job

**Pull request merged**